integration/simple-force-repeated/force-repeated-test.ts

```diff
@@ -0,0 +1,28 @@
+import { Simple, SimpleWithWrappers, StateEnum } from './simple'
```

Comment:
Disclaimer I did more than my fair share of "just copy/paste all of `simple.proto` and go" when making new integrations, but have since tried to get better about making as-small-as-possible-`proto`s for new integration tests.

Granted, you've already got `simple.proto` copy/pasted, but can you prune that back and using a more minimal file/schema?

You may just need a single message with a single repeated field at this point. (Maybe more depending on the `matchJsonParseOutput` use case.)